examples/sysctrl/hello-sysctrl/examples/init_hpc.rs

```diff
@@ -99,22 +100,43 @@ fn main() -> ! {
 
     print_example_name!();
 
+    // Enable SDRAM
+    let ddr_mode = 0x1;
+    let axi_enable = 0x2;
+    sprint!("Enabling SDRAM ddr_mode={:#x}, axi_enable={:#x}...",
+            ddr_mode,
+            axi_enable,
+    );
+    soc_ctrl::sdram_cfg_axi_ddr_mode_mask(ddr_mode);
+    soc_ctrl::sdram_cfg_axi_enable_mask(axi_enable);
+    sprintln!(" done");
+
     for i in 0..5 {
         let addr = HPC_BOOTRAM_ADDR + i * 4;
         sprint!("Writing 0x6f into {:#x}...", addr);
         unsafe { ptr::write_volatile(addr as *mut _, 0x6f) };
         sprintln!(" done");
     }
 
-    let hpc_core_en = 0xf;
+    // Configure execute regions for SDRAM
+    let execute_region_pattern = 0x6fff_ffff;
+    sprint!("Configuring execute regions for SDRAM using pattern: {:#x}...",
+            execute_region_pattern,
+    );
+
+    let hpc = unsafe { pac::Hpc::steal()};
+    hpc.cluster_config()
+        .execute_region_length2()
+        .write(|w| unsafe {w.bits(execute_region_pattern)});
+    sprintln!(" done");
+
+    // Turn on all HPC cores
```

Comment:
Comment is outdated. We turn on one core only.

Reply:
Updated comment to only refer to one core